### ISSUE TITLE
[CHANGE] Training for Procurement Officers isn't Optional

### ISSUE THREAD

**Issue by mgifford** [gc-da11yn/gc-da11yn.github.io#215]
2.3.8.3 Departments must provide learning opportunities on accessibility requirements for employees who procure, develop or implement ICT.
https://a11y.canada.ca/en/standards/standard/#2.3.8.3

The model the USA Federal government built around agile procurement could be emulated for accessibility procurement  - https://www.usds.gov/projects/ditap

There is no reason why all digital procurement over $1 million wouldn't be managed by procurement officers which have been trained on how to research and evaluate accessibility from vendors. That should be progressively implemented so that smaller and smaller contracts are being managed by people who understand and care about accessibility. Can't can't continue to be an afterthought. 

Use of OpenACR would be pretty great too https://gsa.github.io/openacr-editor/ - no need to consider the VPAT in 2022.
